posts/2023-02-20-liberty-tools-eclipse-deep-dive.adoc

```diff
@@ -0,0 +1,118 @@
+
+== Requirements
+
+Liberty Tools for Eclipse IDE is designed to work with link:https://github.com/OpenLiberty/liberty-tools-eclipse/blob/main/docs/user-guide.md#software-requirements[supported versions] of both 
+the IDE for Java EE Developers and the IDE for Java Developers. While Java 17 is needed to run the tool, you can choose a different version of Java to run your application. 
+
+Since Liberty Tools uses the Liberty Maven and Liberty Gradle plugins behind the scenes, these plugins must be configured in your applications `pom.xml` or `build.gradle` files. Liberty Tools will also need to run the Maven or Gradle CLI which can be accomplished by configuring the Maven and Gradle home in the link:https://github.com/OpenLiberty/liberty-tools-eclipse/blob/main/docs/user-guide.md#setting-preferences[tool's preferences], or including a wrapper with the application.
+
+== Managing your applications
+
+Any application in your Eclipse workspace that is configured with the Liberty Maven or Liberty Gradle plugin is automatically enabled to run with Liberty Tools. Actions are available to manage these apps such as `Start`, `Stop`, `Debug` and `Run Tests`, and you can run these actions from either the Liberty Dashboard or the Eclipse "Run As" menu.
+
+=== Liberty Dashboard
+
+The Liberty Dashboard consolidates the Liberty enabled applications in your workspace into a single view. To open the dashboard, click on Open Liberty UFO icon on the top tray. To run an action, simply right click on an app in the list.
+
+[.img_border_light]
+image::/img/blog/liberty-tools-eclipse-dash-menu.png[Liberty Dashboard,width=70%,align="center"]
+
+=== Eclipse Run As menu
+
+Liberty Tools offers the same set of actions in the Eclipse "Run As" menu that appears when you right click on a project in the Project Explorer view.
+
+[.img_border_light]
+image::/img/blog/liberty-tools-eclipse-runas-menu.png[Run As menu,width=70%,align="center"]
+
+=== Starting your app
+
+To start your application in dev mode, select either the `Start` or `Start...` actions. The latter will open a `Run Configurations` dialog box where you can add link:https://github.com/OpenLiberty/ci.maven/blob/main/docs/dev.md#additional-parameters[additional parameters] to dev mode or configure the JRE to use when running the app. When a start action is selected, Liberty Tools opens a `Terminal` tab and starts dev mode. 
+
+[.img_border_light]
+image::/img/blog/liberty-tools-eclipse-run-config.gif[Run Configurations,width=70%,align="center"]
+
+[.img_border_light]
+image::/img/blog/liberty-tools-eclipse-terminal-start.png[Terminal Start,width=70%,align="center"]
+
+NOTE: The `Start in container` action will link:https://github.com/OpenLiberty/ci.maven/blob/main/docs/dev.md#devc-container-mode[start dev mode in a local Docker container]. 
```

Comment:
Maybe link to the PATH setup?

..and requires 'docker' on the PATH? (link to
https://github.com/OpenLiberty/liberty-tools-eclipse/blob/main/docs/user-guide.md#docker